docs/proposals/20250513-propogate-taints.md

```diff
@@ -0,0 +1,625 @@
+---
```

Comment:
Opened a PR here for some minor findings & fixes: https://github.com/nrb/cluster-api/pull/1

Reply:
merged in